### PR TITLE
Support for multiple string arguments to frame helper

### LINKED ISSUE #503: turbo_frame_tag with arrary argument regression
```
<%= turbo_frame_tag [@term_hour.location_id, @term_hour.term.to_param, params[:day_of_week]] do %>
```

With 1.4.0:
```
<turbo-frame id="1_1_sunday">
```

With 1.5.0:

```
<turbo-frame id="1 1 sunday">
```

It seems like #476 broke this behavior.

### PR DESCRIPTION
**Pull request by jcoyne**
This fixes a regression in 1.5.0

Fixes #503

### CODE REVIEW

File: app/helpers/turbo/frames_helper.rb [hunk 1]
```diff
@@ -36,7 +36,7 @@ module Turbo::FramesHelper
   #   <%= turbo_frame_tag(Article.find(1), Comment.new) %>
   #   # => <turbo-frame id="article_1_new_comment"></turbo-frame>
   def turbo_frame_tag(*ids, src: nil, target: nil, **attributes, &block)
```
Comment (seanpdoyle) [inline app/helpers/turbo/frames_helper.rb:38]: Could you add an example to the documentation comment above like the one in https://github.com/hotwired/turbo-rails/pull/505/files#diff-7694cf5e712468388f0d32c6638e9d9508b612f0346ce4beb9093960a757fa59R26-R28?

**Review by seanpdoyle — APPROVED** [review]

File: app/helpers/turbo/frames_helper.rb (hunk 1 above)
Comment (jcoyne) [reply app/helpers/turbo/frames_helper.rb:41]: Yes, I've pulled your example in.

**Comment by jcoyne** [comment]
@seanpdoyle it looks like one of the tests is timing out and I don't have the permissions to restart it.

**Comment by seanpdoyle** [comment]
I don't have permissions either. If you rebase the commit and force push, it'll run again

**Comment by jcoyne** [comment]
@afcapel are you able to restart that one build?

**Comment by swanson** [comment]
Thank you! Ran into this as well when upgrading

**Pull request merged**